dgf_cms/settings.py

```diff
@@ -314,15 +324,25 @@ def dbbackup_cleanup_filter(filename):
     'djangocms_bootstrap4.contrib.bootstrap4_picture',
```

Comment:
`djangocms_bootstrap4.*` needs to be deleted later after the `djangocms_frontend.*` (based on bootstrap 5)migration is done